smokey/src/main/java/frc/robot/subsystems/CameraSubsystem.java

```diff
@@ -12,42 +12,36 @@
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
```

Comment:
Curious to know why this class isn't a 100% change/new?  Did we have a camera subsystem code prior to this?

Reply:
yes, you made a stub in smokey for me at the end of the season to use over the summer.